pages/whats-new/overview.md

```diff
@@ -12,12 +12,20 @@ columns:
   - title: Page views
     source: Pageviews
 redirect_from:
+  - /about/releases/
   - /about-our-work/
+  - /about-our-work/releases
   - /whats-new/
+  - /whats-new/releases
 in_page_nav: false
 ---
 
-{% capture blog %}
+## Release notes
+
+You can find our release notes — summaries of bug fixes, new features, and other updates introduced in each release on our [GitHub releases page](https://github.com/uswds/uswds/releases).
+
+You can subscribe to our GitHub releases by visiting the USWDS [GitHub repo](https://github.com/uswds/uswds/), selecting the `watch` button, selecting `custom`, checking the box for `releases`, and selecting apply.
```

Comment:
@sarah-sch, @finekatie Can you take a look at this copy in this section and let us know if you have any suggestions?

Reply:
Ok, my lack of GitHub is going to show. I can't tell which line 27 above is the change, or if both sections would be part of the page. I saw what was in the preview, and below is my response to that. Apologies if I've confused further!  @sarah-sch , thoughts?
____
You can find summaries of bug fixes, new features, and other updates introduced in each release on our [GitHub releases page](https://github.com/uswds/uswds/releases).

Want to learn about new USWDS releases as they are published? Subscribe now: Visit the USWDS [GitHub repo](https://github.com/uswds/uswds/). Select the watch button, select custom, check the box for releases, and select apply.

Reply:
I combined the ideas so that it followed @finekatie's copy recommendation and @amyleadem's idea to rely on GH instructions on subscribing to releases 👍 

## Release notes

You can find summaries of bug fixes, new features, and other updates introduced in each release on our [GitHub releases page](https://github.com/uswds/uswds/releases).

Want to learn about new USWDS releases as they are published? You can sign up to be notified of new USWDS releases by setting up [watch notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#configuring-your-watch-settings-for-an-individual-repository) in the USWDS repository in GitHub. 

--- 

Resolved in [34e5698](https://github.com/uswds/uswds-site/pull/2318/commits/34e56981c3d8e9dd963ba7d5422777097789db8e)

Reply:
Looks good to me!